Podcast/ProfileMiniHeader.swift

```diff
@@ -12,73 +12,62 @@ import UIKit
 
 class ProfileMiniHeader: UIView {
```

Comment:
They're different views entirely. The internal one was the old profileViewMiniHeader